### PR TITLE
[datapackage][m]: created datapackage.json and modified data

### PR DESCRIPTION
**Pull request by Mikanebu**
* Created datapackage.json
* Modified data: "NA" value is not supported in number type (according to the specifications https://pre-v1.frictionlessdata.io/json-table-schema/#number)

### CODE REVIEW

File: datapackage.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
+{
+  "name": "corruption-perceptions-index",
+  "resources": [
+    {
+      "path": "data/cpi.csv",
+      "name": "data",
+      "schema": {
+        "fields": [
+          {
+            "description": "",
```
Comment (zelima) [inline datapackage.json]: Why do we need empty descriptions? this is not required field

File: datapackage.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            "format": "default",
```
Comment (zelima) [inline datapackage.json]: Same here - why do we specify format to default? it will automatically be done right?

File: datapackage.json [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            "type": "string",
+            "name": "Jurisdiction",
+            "title": ""
```
Comment (zelima) [inline datapackage.json]: again, non required field with empty string, let's remove

File: datapackage.json (hunk 1 above)
Comment (Mikanebu) [inline datapackage.json]: Good point

**Comment by Mikanebu** [comment]
Good review @zelima . I will create 2 separate PR for the datapackage.json and generations script

**Comment by zelima** [comment]
@Mikanebu Do not create new PR. just add new commits here. We will squash and merge, it was just a tip for future ;)

**Pull request merged**